src/components/ContentList/ContentListItem.vue

```diff
@@ -48,6 +45,16 @@ export default {
 			} catch (e) {
 				return 'grey'
 			}
+		},
+		avatarUrl() {
+			return `url(${this.contact.url}?photo)`
+		},
+		initial() {
+			if (this.avatarUrl) {
```

Comment:
A contact always have an url. You cannot assume the contact have a picture or not.
If a contact does NOT have an url, that means it's a local contact (present on the store only and not created on the server **yet**) that have not yet been pushed to the server. So in that case, yes we can assume it doesn't have an url. Though I'll suggest checking the `contact.dav` property to check if this is a local contact or not.